src/applications/simple-forms/21-0966/config/form.js

```diff
@@ -1,20 +1,83 @@
 import footerContent from 'platform/forms/components/FormFooter';
+import environment from '@department-of-veterans-affairs/platform-utilities/environment';
 import manifest from '../manifest.json';
 
+import transformForSubmit from '../../shared/config/submit-transformer';
 import IntroductionPage from '../containers/IntroductionPage';
 import ConfirmationPage from '../containers/ConfirmationPage';
 import getHelp from '../../shared/components/GetFormHelp';
+import preparerIdentification from '../pages/preparerIdentification';
+import veteranBenefitSelection from '../pages/veteranBenefitSelection';
+import preparerPersonalInformation from '../pages/preparerPersonalInformation';
+import preparerIdentificationInformation from '../pages/preparerIdentificationInformation';
+import mailingAddress from '../pages/mailingAddress';
+import phoneAndEmailAddress from '../pages/phoneAndEmailAddress';
+import {
+  preparerIsVeteran,
+  preparerIsSurvivingDependant,
+  preparerIsThirdParty,
+  preparerIsThirdPartyToTheVeteran,
+  preparerIsThirdPartyToASurvivingDependant,
+  benefitSelectionStepperTitle,
+  personalInformationStepperTitle,
+  contactInformationStepperTitle,
+} from './helpers';
+import survivingDependantBenefitSelection from '../pages/survivingDependantBenefitSelection';
+import veteranPersonalInformation from '../pages/veteranPersonalInformation';
+import veteranIdentificationInformation from '../pages/veteranIdentificationInformation';
+import preparerName from '../pages/preparerName';
 
+// mock-data import for local development
+import testData from '../tests/e2e/fixtures/data/minimal-test.json';
+import relationshipToVeteran from '../pages/relationshipToVeteran';
+
+const mockData = testData;
+
+/** @type {FormConfig} */
 const formConfig = {
   rootUrl: manifest.rootUrl,
   urlPrefix: '/',
-  // submitUrl: `${environment.API_URL}/simple_forms_api/v1/simple_forms`,
+  submitUrl: `${environment.API_URL}/simple_forms_api/v1/simple_forms`,
   submit: () =>
-    Promise.resolve({ attributes: { confirmationNumber: '123123123' } }),
+    Promise.resolve({
+      confirmationNumber: '123123123',
```

Comment:
non-blocking as long as form is not intended for launch yet: just noting this part as something needing to be changed before form is ready 👀

Reply:
I've just commented it all out for now.